R/ResultFlagsDependent.R

```diff
@@ -94,22 +94,22 @@ InvalidFraction <- function(.data, clean = TRUE, errorsonly = FALSE) {
   if (any(c("Nonstandardized", "Invalid") %in%
           unique(check.data$TADA.SampleFraction.Flag)) == FALSE) {
     if (errorsonly == FALSE) {
-      print("No changes were made, because we did not find any invalid fraction/characteristic combinations in your dataframe")
-      .data = OrderTADACols(.data)
-      return(.data)
+      print("All characteristic/fraction combinations are valid in your dataframe. Returning input dataframe with TADA.SampleFraction.Flag column.")
```

Comment:
Suggestion: Returning input dataframe with added TADA.SampleFraction.Flag column for tracking.